docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-suspicious-dll-loaded-for-persistence-or-privilege-escalation.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[prebuilt-rule-0-16-2-suspicious-dll-loaded-for-persistence-or-privilege-escalation]]
+=== Suspicious DLL Loaded for Persistence or Privilege Escalation
+
+Identifies the loading of a non Microsoft signed DLL that is missing on a default Windows install (phantom DLL) or one that can be loaded from a different location by a native Windows process. This may be abused to persist or elevate privileges via privileged file write vulnerabilities.
```

Comment:
```suggestion
Identifies the loading of a non Microsoft-signed DLL that is missing on a default Windows install (phantom DLL) or one that can be loaded from a different location by a native Windows process. This may be abused to persist or elevate privileges via privileged file write vulnerabilities.
```